docs/scripts/util/crawlAllLinksFromAllPages.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+import { runArrayPromiseInOrder } from '.';
+
+export async function crawlAllLinksFromAllPages(allPages: string[]) {
+  type AllPagesPaths = Map<
+    string,
+    {
+      pageUrl: string;
+      links: { href: string; tagName: string; tagText: string }[];
+    }
+  >;
+  const allPagesPaths: AllPagesPaths = new Map();
+  await runArrayPromiseInOrder(allPages, checkSitemapPath);
```

Comment:
Does this really need to run in a specific order?

Reply:
Yes, otherwise 1) links log and table at the bottom would be in different order each time, more importantly 2) it cause memory leak,  use up TCP, sockets, or file descriptor, etc...

e.g.
> MaxListenersExceededWarning: Possible EventEmitter memory leak detected